src/scripts/main.js

```diff
@@ -1,3 +1,8 @@
 'use strict';
 
-// write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+wall.style.display = 'grid';
+wall.style.placeItems = 'center';
+spider.style.position = 'inherit';
```

Comment:
The 'position' property is set to 'inherit', which means that the spider element will inherit the position property from its parent. If the parent does not have a position set, this can lead to unexpected behavior. It would be more reliable to set a specific position value that suits the layout requirements.